### ISSUE TITLE
Alerts/ Support Multipoint

### ISSUE THREAD

**Issue by ximenabb** [digidem/comapeo-mobile#874]
The front-end code ([PR#844](https://github.com/digidem/comapeo-mobile/pull/844)) supports displaying alerts of type Point, LineString, MultiLineString, Polygon and MultiPolygon. 
It does not support `MultiPoint`

ToDo

- [ ] Add support of `MultiPoint`

**Comment by EvanHahn** [comment]
Haven't tested it, but I _think_ this is all that's required in `src/frontend/screens/MapScreen/MapLayers/RemoteDetectionAlertsLayer.tsx`:

```diff
@@ -31,6 +31,7 @@ export const RemoteDectionAlertsMapLayer = () => {
             'Point',
             'MultiLineString',
             'MultiPolygon',
+            'MultiPoint',
           ],
           ['has', 'metadata.alert_type'],
           ['has', 'month_detec'],
@@ -59,7 +60,7 @@ export const RemoteDectionAlertsMapLayer = () => {
       {/* Circle Layer for Points */}
       <MapboxGL.CircleLayer
         id="comapeo-alerts-point"
-        filter={['==', '$type', 'Point']}
+        filter={['in', '$type', 'Point', 'MultiPoint']}
         style={{
           circleRadius: 5,
           circleColor: '#FF0000',
```

**Comment by ErikSin** [comment]
We could do it the way that Evan has recommended, the only thing is that it would be in-distinguishable from normal points. Does the CMI team want them styled differently (aka Not red dots) from the other alerts?

**Comment by ximenabb** [comment]
@ErikSin they can be the same style (red dot) as other alerts

**Comment by gmaclennan** [comment]
Yes, generally there should be no need to style `Multi` geometry variants differently than their non-multi versions, e.g. `Line` and `MultiLine` should be the same, `Polygon` and `MultiPolygon` should be the same, and `Point` and `MultiPoint` should be the same.
